### PR TITLE
Report standardization/ add tool icons

### PR DESCRIPTION
**Pull request by eadams17**
## WHAT
add tool icons to student overview table

## WHY
we want to render icons for each of the different tools

## HOW
add these icons to our assets and then conditionally render them in the applicable report

### Screenshots
(If applicable. Also, please censor any sensitive data)
Screen Shot 2024-07-10 at 5 21 30 PM

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)

### What have you done to QA this feature?
(Provide enough detail that a reviewer could assess whether additional QA should be done. For larger projects, additionally use the Engineer Feature Testing Notion template. Review Guidelines if needed: https://www.notion.so/quill/Github-PR-QA-Guidelines-49e99fc965654ceeb8c6249bd9d181d7)
verified on staging that the icons render as expected

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | yes
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? | yes

### CODE REVIEW

File: services/QuillLMS/app/assets/stylesheets/pages/progress_reports/diagnostic_student_responses_index.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
       th {
         height: 100%;
         &:not(:last-child) {
-          border-right: 1px solid #CFCFCF;
+          border-right: 1px solid #CFCFCF !important;
```
Comment (anathomical) [inline services/QuillLMS/app/assets/stylesheets/pages/progress_reports/diagnostic_student_responses_index.scss]: Should this color be a variable if we have one for it?

File: services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/student_overview_table.jsx [hunk 2]
```diff
@@ -53,6 +53,38 @@ export default class StudentOveriewTable extends React.Component {
     }
   }
 
+  activityClassificationIcon(id) {
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/student_overview_table.jsx:56]: This feels like maybe it should be a Shared function, but this seems reasonable for now given that we're only using it in this one place.

**Review by anathomical — APPROVED** [review]
A couple of comments, but I think this could go out as-is.

File: services/QuillLMS/app/assets/stylesheets/pages/progress_reports/diagnostic_student_responses_index.scss (hunk 1 above)
Comment (eadams17) [reply services/QuillLMS/app/assets/stylesheets/pages/progress_reports/diagnostic_student_responses_index.scss]: yes, but this actually shouldn't be in this PR-- removing this file now

**Pull request merged**